_docs/data-model/v1/device-meta.md

```diff
@@ -11,9 +11,96 @@ The "type" of a deviceMeta event is defined by the `subType` field.
 
 Current DeviceMeta events are
 
+* alarm
+* prime
+* reset
 * status
 * calibration
 
+## Alarm
+
+An alarm event is used to represent a user-facing alarm from an insulin pump. An alarm event looks like
+
+~~~json
+{
+  "type": "deviceMeta",
+  "subType": "alarm",
+  "alarmType": type_of_alarm_or_other,
```

Comment:
seems like gitHub is now calling out errors in things marked as JSON. Maybe we should put these in quotes (and in other places where it occurs)